docs/development/extensions-core/kubernetes.md

```diff
@@ -31,11 +31,10 @@ Apache Druid Extension to enable using Kubernetes API Server for node discovery
 
 To use this extension please make sure to  [include](../../configuration/extensions.md#loading-extensions) `druid-kubernetes-extensions` in the extensions load list.
 
-This extension works together with HTTP based segment and task management in Druid. Consequently, following configurations must be set on all Druid nodes.
+This extension works together with HTTP-based segment and task management in Druid. Consequently, following configurations must be set on all Druid nodes.
 
 `druid.zk.service.enabled=false`
 `druid.serverview.type=http`
-`druid.coordinator.loadqueuepeon.type=http`
```

Comment:
There's one more reference in `integration-tests/k8s/tiny-cluster.yaml:    druid.coordinator.loadqueuepeon.type=http` which can also be removed.

Reply:
Thanks for catching this, @abhishekrb19 . There are a couple more references in the code that I plan to tackle in a separate PR. Keeping this one only for the docs changes.